src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Random should be class field same as in Lottery class in order not to create new object each time method is called